server/users.js

```diff
@@ -606,8 +609,9 @@ class User extends Chat.MessageContext {
 	 */
 	getIdentityWithStatus(roomid = '') {
 		const identity = this.getIdentity(roomid);
-		if (!this.status) return identity;
-		return `${identity}@${this.status}`;
+		const statusSymbol = this.statusType === 'busy' ? '!' : this.statusType === 'idle' ? '?' : '';
```

Comment:
This is the new protocol, and would need a client change before it could be implemented.

Reply:
You know, if you move the protocol changes to a different PR, this could get merged a lot sooner. It should probably be done in a different PR anyway.

Reply:
```suggestion
		const statusSymbol = this.statusType === 'busy' ? '!(Busy) ' : this.statusType === 'idle' ? '!(Idle) ' : '';
```